docs/Language Feature Status.md

```diff
@@ -0,0 +1,31 @@
+# Language Feature Status
+
+## C# 7.0 and VB 15
+
+| Feature | Branch | State | Owners |
```

Comment:
Send me the document where the current champions are set and I will add it to the table.